### PR TITLE
Updated Onnxruntime quantization notebook

### PR DESCRIPTION
**Pull request by philschmid**
# What does this PR do?
 
Updates Optimum notebook to support the new quantization methods

### CODE REVIEW

File: examples/text_classification_quantization_ort.ipynb [hunk 1]
```diff
@@ -48,14 +48,14 @@
...
-   "execution_count": 2,
+   "execution_count": 18,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "1.1.0\n"
+      "1.4.0.dev0\n"
```
Comment (echarlaix) [inline examples/text_classification_quantization_ort.ipynb:58]: Should we also add a "Make sure your version of 🤗 Optimum is at least 1.4.0 "

**Review by echarlaix — APPROVED** [review]
Thanks for taking care of this @philschmid, it looks great

File: examples/text_classification_quantization_ort.ipynb (hunk 1 above)
Comment (philschmid) [reply examples/text_classification_quantization_ort.ipynb:58]: lets re run it after the launch again?

**Comment by lewtun** [comment]
Shall we merge this?

**Pull request merged**